sdk/keyvault/azsecrets/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.2.1 (Unreleased)
 
 ### Features Added
+* Uses a challenge authentication policy
```

Comment:
I think this would end up in customer-facing release notes. From a customer's perspective the feature is something like 
```suggestion
* Clients can now connect to Key Vaults in any cloud
```